src/views/Homepage.js

```diff
@@ -61,21 +135,19 @@ const Homepage = ({ handleIsBrandDark, handleIsNavbarDark }) => {
 
   return (
     <StyledMain>
-      <StyledHero src="/img/photos/bridge_in_forest.jpg">
-        <CameraOverlay />
+      <StyledHero src="/img/photos/homepage_photo">
```

Comment:
Having this one prop saves a few bytes from the final bundle, but my preference would be two separate props for each `src` (desktop and mobile), and the `src` values include the file extension in case you need a transparent background via `.png` in mobile/desktop as opposed to a `.jpg` for more lossy compression.

Reply:
Just a suggestion. Not necessary to fix.